### PR TITLE
add Interpretation.show_at method

### PR DESCRIPTION
**Pull request by vrodriguezf**
This is basically the code added (by @muellerzr)

```python
def __getitem__(self:Interpretation, idxs):
    "Get the inputs, preds, targets, decoded outputs, and losses at `idx`"
    if not is_listy(idxs): idxs = [idxs]
    attrs = 'inputs,preds,targs,decoded,losses'
    res = L([getattr(self, attr)[idxs] for attr in attrs.split(',')])
    return res

@patch
def show_at(self:Interpretation, idx:int, **kwargs):
    inp, _, targ, dec, _ = self[idx]
    dec, targ = L(dec, targ).map(Self.unsqueeze(0))
    self.dl.show_results((inp, dec), targ, **kwargs)
```

I just showcased it following in a nb following the format of this repo. The methods are exported to the module `basics.interp.py`

### CODE REVIEW

**Comment by muellerzr** [comment]
Thanks @vrodriguezf (finally able to sit down and take a look). It would be _nice_ if we could explain just what our `show_at` code is doing internally, as we both know there's much to be learned through those simple lines of code! Would it be reasonable to ask you to make that a bit more verbose? 

If that's a bit out of your ballpark, I can expand upon it later, just wanted to give you the opportunity first!

Also, when building make sure you have the latest pip nbdev (1.1.11) and do `nbdev_build_lib`. This _should_ fix your build errors.

Thank you again! :)

File: wwf/basics/interp.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@patch
+def __getitem__(self:Interpretation, idxs):
+    "Get the inputs, preds, targets, decoded outputs, and losses at `idx`"
+    if not is_listy(idxs): idxs = [idxs]
+    attrs = 'inputs,preds,targs,decoded,losses'
+    res = L([getattr(self, attr)[idxs] for attr in attrs.split(',')])
+    return res
+
+@patch
+def show_at(self:Interpretation, idx:int, **kwargs):
```
Comment (muellerzr) [inline wwf/basics/interp.py:15]: Perhaps include a docstring here. Also would we want to have them be able to show multiple index's? Or is just one good enough. I'm leaning towards a situation similar to `__getitem__` where we check for a list of index's

**Review by muellerzr — CHANGES_REQUESTED** [review]
Also let's try to rename the notebook `05_interp.show_at`, this way anything else tying to interp goes to `interp.x`. I think I'm fine with the basics, I like having that there and may move over the `lr_finder` improvements to that as well! It's a good way to section out what's an improvement vs an implementation.

File: wwf/basics/interp.py (hunk 1 above)
Comment (vrodriguezf) [reply wwf/basics/interp.py:15]: Sure! I assume that the creation of the grid for showing multiple items would be delegated to `show_results`, is that correct?

**Comment by vrodriguezf** [comment]
Will do @muellerzr , thanks for the review!

**Comment by muellerzr** [comment]
Correct. For it to work I believe you just need to modify idxs to become a
list (similar to the first two lines in the other function). Otherwise the
rest should still be the exact same

**Comment by vrodriguezf** [comment]
Hi! Sorry for the delay, it was a crazy week...I updated to nbdev 1.1.11 and added some explanations to the code of `show_at`. It's still failing though!

File: wwf/basics/interp.py (hunk 1 above)
Comment (muellerzr) [reply wwf/basics/interp.py:15]: Yes that would be (sorry for just now getting to that)

**Comment by muellerzr** [comment]
Awesome work @vrodriguezf! It's not failing for me, so don't worry about that :) Thank you for the hard work and an awesome post!

**Pull request merged**